sdk/oracle/Azure.ResourceManager.OracleDatabase/samples/Generated/Samples/Sample_CloudVmClusterResource.cs

```diff
@@ -141,7 +141,7 @@ public async Task AddVms_AddVMsToVMCluster()
         public async Task GetPrivateIPAddresses_ListPrivateIPAddressesForVMCluster()
         {
             // Generated from example definition: specification/oracle/resource-manager/Oracle.Database/stable/2023-09-01/examples/vmClusters_listPrivateIpAddresses.json
-            // this example is just showing the usage of "CloudVmClusters_ListPrivateIPAddresses" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "CloudVmClusters_ListPrivateIpAddresses" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `Ip`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/oracle/Oracle.Database.Management/operations/exadata/vmclusterOperations.tsp#L43